fed/proxy/barriers.py

```diff
@@ -51,6 +51,39 @@ def set_receiver_proxy_actor_name(name: str):
     _RECEIVER_PROXY_ACTOR_NAME = name
 
 
+def set_proxy_actor_name(job_name: str,
+                         use_global_proxy: bool,
```

Comment:
`use_global_proxy` can be omitted ? Job_name is enough to identify the proxy actor name, since default job name means `use_global_proxy=true` already.

Reply:
But `job_name` is not Intentionally used for the flag of customizing proxy actor, what if the user sets `job_name` but doesn't want to start a separate proxy actor?

Reply:
👌🏻, it sounds meaningful.